platform/admin/ui/src/main/webapp/templates/configuration/textTypeList.handlebars

```diff
@@ -12,7 +12,20 @@
  **/
  --}}
  <td>
-    <input type='text' name='value' class="{{id}} metatype" id='{{id}}' value='{{value}}'>
+     {{#if optionValues}}
+         {{#is optionValues.length 0}}
+             <input type="{{#is type 1}}text{{/is}}{{#is type 5}}text{{/is}}{{#is type 6}}text{{/is}}{{#is type 7}}text{{/is}}{{#is type 8}}text{{/is}}{{#is type 9}}text{{/is}}{{#is type 10}}text{{/is}}{{#is type 12}}password{{/is}}{{#is type 2}}number{{/is}}{{#is type 3}}number{{/is}}{{#is type 4}}number{{/is}}" class="metatype" name="value" value="{{value}}">
```

Comment:
It is invalid without a type, so it shouldn't need any sort of fallback (we don't currently have a fallback for the rest of the fields in the admin console either). I can add a comment though.